components/match2/wikis/wildrift/match_legacy.lua

```diff
@@ -0,0 +1,219 @@
+				game.extradata[newKey] = item
+			end
+		end
+		game.extradata.team1side = extradata.team1side
+		game.extradata.team2side = extradata.team2side
+
+		game.extradata = mw.ext.LiquipediaDB.lpdb_create_json(game.extradata)
+
+		-- Other stuff
+		game.opponent1 = match.opponent1
+		game.opponent2 = match.opponent2
+		game.opponent1flag = match.opponent1flag
+		game.opponent2flag = match.opponent2flag
+		game.date = match.date
+		local winner = tonumber(game.winner) or 0
+		game.opponent1score = winner == 1 and 1 or 0
+		game.opponent2score = winner == 2 and 1 or 0
+		local res = mw.ext.LiquipediaDB.lpdb_game(
+			'legacygame_' .. match2.match2id .. gameIndex,
+			game
+		)
+		games = games .. res
+	end
+	return games
+end
+
+function MatchLegacy.storeMatchSMW(match, match2)
+	local data = {
+		'legacymatch_' .. match2.match2id,
+		'is map number=1',
+		'has team left=' .. (match.opponent1 or ''),
+		'has team right=' .. (match.opponent2 or ''),
+		'Has map date=' .. (match.date or ''),
+		'Has tournament=' .. mw.title.getCurrentTitle().prefixedText,
+		'Has tournament tier=' .. (match.liquipediatier or ''),
+		'Has tournament name=' .. Logic.emptyOr(
+			match.tickername,
+			match.name,
+			Variables.varDefault('tournament_name', mw.title.getCurrentTitle().prefixedText)
+		),
+		'Has tournament icon=' .. Variables.varDefault('tournament_icon', ''),
+		'Has winner=' .. (match.winner or ''),
+		'Has team left score=' .. (match.opponent1score or '0'),
+		'Has team right score=' .. (match.opponent2score or '0'),
+		'Has exact time=' .. (Logic.readBool(match.dateexact) and 'true' or 'false'),
+		'Is finished=' .. (Logic.readBool(match.finished) and 'true' or 'false'),
+		'Has teams=' .. (match.opponent1 or ''),
+		'Has teams=' .. (match.opponent2 or ''),
+		'Has teams name=' .. (match.opponent1 or ''),
+		'Has teams name=' .. (match.opponent2 or ''),
+		'Has calendar description=- ' .. match.opponent1 .. ' vs ' .. match.opponent2 .. ' on ' .. match.date,
+	}
+
+	local streams = match.stream or {}
+	streams = Json.parseIfString(streams)
+	for key, item in pairs(streams) do
+		table.insert(
+			data,
+			'Has match ' .. key .. '=' .. item
+		)
+	end
+
+	local extradata = match.extradata or {}
+	extradata = Json.parseIfString(extradata)
+	for key, item in pairs(extradata) do
+		if String.startsWith(key, 'vodgame') then
+			table.insert(
+				data,
+				'Has match ' .. key .. '=' .. item
+			)
+		end
+	end
+
+	local mvpTable = mw.text.split(extradata.mvp or '', ',')
+	local mvpTeamsTable = mw.text.split(extradata.mvpteam or '', ',')
+	if String.isNotEmpty(mvpTable[1]) then
+		for key, item in pairs(mvpTable) do
+			local mvp = mw.text.trim(item)
+			local mvpTeam = mw.text.trim(mvpTeamsTable[key] or '')
+			if Logic.isNumeric(mvpTeam) then
+				mvpTeam = match['opponent' .. mvpTeam]
+			else
+				mvpTeam = mvpTeam or ''
+			end
+			table.insert(
+				data,
+				'Has mvp ' .. key .. '=' .. mvp .. '§§§§' .. mvpTeam .. '§§§§0'
```

Comment:
i assume `§§§§` shouldn't be there?

Reply:
it is how they currently store it in smw hence we need it to be stored like that again to not break stuff
i am not happy about that format either, but it is what it is